### ISSUE TITLE
Spanish translation

### ISSUE THREAD

**Issue by Preyesianyeha** [mimoguz/tripeaks-gdx#4]
I want to help to transtalate this project into spanish, so if you could tell me what i must do, then i'll do!

**Comment by mimoguz** [comment]
Thanks!

It's not very complicated really. This is the resource:

    start=Start
    fromStack=You took {0,choice,0#no cards|1#one card|1<{0,number,integer} cards} from the stack.
    usedUndo=You {0,choice,0#didn't use undo at all|1#used undo once|2#used undo twice|2<used undo {0,number,integer} times}.
    longestChain=Your longest chain was {0,choice,0#zero cards|1#one card|1<{0,number,integer} cards}.
    newGame=Start new game
    newGameShort=New game
    exit=Exit game
    return=Return to game
    darkTheme=Use dark theme
    showAll=Show all cards
    won=You won!

The parts needs to be translated are on the right of the equals signs. Most of them are straightforward, except three. Three of them have calculated parts. Let's go over one of them, fromStack:

* You took: This part will be taken as is, no calculation.
* { : Starts the calculated part
* 0, : Calculation can take multiple parameters, indexed starting from 0. Use the 0th parameter here.
* choice, : The value of the 0th parameter will be used to choose one of the possible options.
* 0#no cards : If 0th parameter's value is 0, choose "no cards".
* |1#one card : Or if it's 1, choose "one card"
* |1< : Or if 1 smaller than the value (the value is bigger than 1) then...
* {0,number,integer}: Another calculated part. This one is simple. It says 0th parameter is an integer number, just give it back.
* cards : Will come after the number.
* } : Calculated part is finished
* from the stack : As is.

Calculated parts of course can be moved, choices can be removed if they are not used, or more choices can be added (usedUndo has another choice for 2 for example).

Some examples:
* If 0th parameter is 0, the result will be "You took **no cards** from the stack."
* If 0th parameter is 1, the result will be "You took **one card** from the stack."
* If it is 5, the result will be "You took **5 cards** from the stack."

Does that make sense? Sorry, English is not my first language.

**Comment by Preyesianyeha** [comment]
start=Empezar
fromStack=Tomaste {0,choice,0#cero cartas|1#una carta|1<{0,number,integer} cartas} de la baraja.
usedUndo={0,choice,0#No usaste deshacer en absoluto|1#Usaste deshacer una vez|2#Usaste deshacer dos veces|2<Usaste deshacer {0,number,integer} veces}.
longestChain=Tu cadena de cartas mas larga fue de {0,choice,0#cero cartas|1#una carta|1<{0,number,integer} cartas}.
newGame=Empezar una nueva partida
newGameShort=Nueva partida
exit=Salir del juego
return=Regresar al juego
darkTheme=Usar tema oscuro
showAll=Mostrar todas las cartas
won=¡Ganaste!

----------------------------------------
This should do, it was fun to do this, if u need anything else please make me now and i'll be there, oh, and in the  third line there's no text at the beginning, because often times in spanish, we don't say the exact subject, we just write the phrase around the subject in mind, so if that were to cause any troble, tell me and i'll change it to add a subject.
(And don't worry about your language, english isn't my first language either, but i did understand you clearly :D)

**Comment by mimoguz** [comment]
If you don't mind, can you translate these too:

* A tri peaks solitaire game.
* An open-source tri peaks solitaire game.
* Currently only has randomly generated games.
* Pre-solved games option is on the roadmap.
* Deal *[a card]*
* Undo
* Menu

These are for the store page, not for the game itself.

Also, question:
I publish this on Play Store too (as is, free, no ads or anything, store page links to this repo). Is that OK for you?

Thank you very much!

**Comment by Preyesianyeha** [comment]
- Un juego de solitario tri peaks.

- Un juego de código-abierto de solitario tri peaks.

- Por ahora solo cuenta con juegos generados aleatoriamente.

- Una opción de juegos pre-resueltos está planeada para el futuro.

- Repartir [una carta]

- Deshacer

- Menú

------------------------------------------------------------------------------
There you go, i hope you like it. :D
Regarding your question: Go ahead dude!, as you are not adding none of that BS, the game is perfect, and if you can get more recognition there, that'd be even better!
(Also i added accents (the little diagonal line above some vowels) to the words that used one, but if you need to get rid of them, just replace them with their normal form).

**Comment by mimoguz** [comment]
Thanks! Accents are no problem. You can add them to in-game text too. 

I'll test and push the changes this weekend. But I think I already missed this week's F-Droid update, so I'll create the release next week.

**Comment by Preyesianyeha** [comment]
Ok then, don't worry about rushing, if u need take ur time!

**Comment by mimoguz** [comment]
Done and done 👍

Before closing the issue though: Does this translation work for both Spain and Latin America? There are some notable differences as far as I understand.

**Comment by Preyesianyeha** [comment]
Yeah, between latin and spain (castellano), there are big differences, however, i used like a middle point, kind of a neutral spanish were the accent doesn't matter, so there should be no problem with that!

**Comment by mimoguz** [comment]
Hi.

I'm readying a new release with a few new features, so I need new strings. I appreciate your help on this.

## New strings

* No more moves! ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png), displayed when the game is stalled)
* Undo last move ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_stalled_dialog.png))
* Save ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
* Cancel  ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png)) 
* Start with an empty discard ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png), [result](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/empty_discard.png))
* New game layout: ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_options_dialog.png))
* Options ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_menu.png)

### Layout options
* Three peaks
* Two peaks and a funnel ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_inverted_2nd_layout.png))
* Diamonds ([Example image](https://raw.githubusercontent.com/mimoguz/tripeaks-gdx/0.5/screenshots/b_0_5_diamonds_layout.png))

### Game description
* An open-source tri peaks solitaire game with multiple layout options.

Thanks!

**Comment by ghost** [comment]
Hi, I hope I'm not intruding.
I can translate that:

- ¡No hay más movimientos! 
- Deshacer el último movimiento
- Guardar
- Cancelar
- Empezar con un descarte vacío
- Disposición de la nueva partida:
- Opciones

Opciones de diseño (layout options)

- Tres picos
- Dos picos y un embudo
- Diamantes

Descripción del juego (game description)
Un juego de solitario tri peaks de código abierto con varias opciones de diseño.

**Comment by mimoguz** [comment]
Thanks!

**Comment by mimoguz** [comment]
Included in new release

**Comment by mimoguz** [comment]
Hello again.

I want to make some small changes for the next release. Could you have look at these:

* Exit   (Before: "Exit **game**")
* Start with an empty discard **pile**   (Before: "Start with an empty discard")
* Two peaks and a **valley**  (Before: "Two peaks and a **funnel**")
* Layout for new games   (Before: "New game layout")
* Your longest chain **had** ...  (Before: "Your longest chain **was** ...")

Spanish versions may be already OK though, some (most) of them are because of my rather poor English. 🙂

Thanks.

**Comment by ghost** [comment]
Hi, I'll help. 
By the way, don't worry, I actually don't speak English well. 
Well, the translations:

- Salir (Before: "Salir del juego")
- Empezar con una pila de descarte vacía (Before: "Empezar con un descarte vacío") 
- Dos picos y un valle (Before: "Dos picos y un embudo")
- Disposición para nuevas partidas (Before: "Disposición de la nueva partida") 
- Tu cadena más larga tenía ... (Before. (literally): "Tu cadena más larga fue [de] ...". (In game): " Tu cadena de cartas más larga fue de ...")

Hey, by the way, I really like your game. 👍

**Comment by mimoguz** [comment]
👍

> Hey, by the way, I really like your game.

Thanks!

**Comment by ghost** [comment]
You are welcome!

**Comment by mimoguz** [comment]
Included in the release 0.51.

**Comment by mimoguz** [comment]
Hi. I want to add one last thing before calling this 1.0. Could you please check the [issue #14](https://github.com/mimoguz/tripeaks-gdx/issues/14)? Thanks.

**Comment by mimoguz** [comment]
[New strings](https://github.com/mimoguz/tripeaks-gdx/issues/27)
